index.html

```diff
@@ -20,7 +20,7 @@ <h1 class="alt-h1 mb-2 text-white">How agencies build&nbsp;software</h1>
     <div class="overflow-hidden">
       <p>GitHub is now Federal Risk and Authorization Management Program (FedRAMP) authorized.
       </p>
-      <a href="{{"/fedramp/" | relative_url}}">Learn More {% octicon chevron-right height:18 class:"d-inline fill-blue ml-1" %}</a>
+      <a href="{{"/fedramp/" | relative_url}}" aria-label="Learn more about GitHub is now Federal Risk and Authorization Management Program (FedRAMP) authorized">Learn More {% octicon chevron-right height:18 class:"d-inline fill-blue ml-1" %}</a>
```

Comment:
```suggestion
      <a href="{{"/fedramp/" | relative_url}}">Learn More {% octicon chevron-right height:18 class:"d-inline fill-blue ml-1" %}</a>
```

Looking again I'm not sure how this would be valuable for accessibility, seems like a SEO thing, so swapping back original.